core/vm/contracts_suave.go

```diff
@@ -196,3 +197,20 @@ func (s *suaveRuntime) doHTTPRequest(request types.HttpRequest) ([]byte, error)
 	}
 	return data, nil
 }
+
+func (s *suaveRuntime) newBuilder() (string, error) {
```

Comment:
would be interesting to think about two kettles sharing the same session ID and doing a `newBuilder(existingSessionID string)` or `connectBuilder`. We don't need to consider for this work just mentioning if it inspires anything again!

Reply:
permisisons around this has some caution signs for me, you could mess with someones simulation if you knew their session ID if done naively